Path on windows bug due to string template

Hi !
I have a project where we have to use Angular 1.x, and I thought that it could be really nice to have some hmr like in 2.x, so i came across your loader.

I'm working on windows, and I had an error like this:
```
ERROR in ./app/app.js
Module not found: Error: Can't resolve 'C:UsersSomeoneSomewhere
ode_modulesangular-hot-loaderlibangular-hot-loader' in [...]
```

For some reason, the `\` in windows path were acting as escape characters. After some research inside your code, I found out that this was because you are using a string template in `loader.js` around line 39:

```js
prependText = `
    if (module.hot) {
      module.hot.accept();
      var hotAngular = require('${require.resolve('./angular-hot-loader')}')(${JSON.stringify(config)});
    }
  `;
```

So basically `require.resolve(...)` resolves the path correctly (i.e. with windows' `\`), but then once inserted in the string template as a string (inside quotes `'`), all the `\` become escape characters and screw things up.
Changing the previous piece of code by this one:

```js
var resolved = require.resolve('./angular-hot-loader').replace(/\\/g, '\\\\');

  prependText = `
    if (module.hot) {
      module.hot.accept();
      var hotAngular = require('${resolved}')(${JSON.stringify(config)});
    }
  `;
```
therefore solved the problem.

If you're OK with this I'd like to open a pull request with the fix above.
Thanks !

--- a/lib/loader.js
+++ b/lib/loader.js
@@ -36,11 +36,12 @@ module.exports = function(source, map) {
   var prependText;
   var node;
   var result;
+  var resolved = require.resolve('./angular-hot-loader').replace(/\\/g, '\\\\');
 
   prependText = `
     if (module.hot) {
       module.hot.accept();
-      var hotAngular = require('${require.resolve('./angular-hot-loader')}')(${JSON.stringify(config)});
+      var hotAngular = require('${resolved}')(${JSON.stringify(config)});
     }
   `;
 

